### PR TITLE
Truncate circle button text after 24 characters

### PR DESCRIPTION
**Pull request by jhanstra**
See https://github.com/UW-Madison-DoIT/angularjs-portal/pull/351

### CODE REVIEW

**Comment by timlevett** [comment]
Hmmmm, I like the feature. I'm sure someone will use `circle-button` outside of the scope of a widget, where they may not want the title to be truncated, and that could be frustrating for developers. Maybe make the truncate number a variable? Then for our case in `lol` we just set it to 24?

**Comment by jhanstra** [comment]
Good idea. Ask and you shall receive.

File: src/main/webapp/portal/misc/partials/circle-button.html [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
 <a ng-href="{{ cbDisabled ? '#' : href}}" target="{{cbDisabled ? '' : target}}">
   <div class="btn btn-primary rounded icon-button text-right" ng-class="{ 'btn-disabled' : cbDisabled}"> 
     <i class="fa {{faIcon}} fa-2x"></i>
   </div>
 </a>
 <p ng-show='title'>
-  {{title}}
-</p>
\ No newline at end of file
+  {{title | truncate:trunclen}}
```
Comment (apetro) [inline src/main/webapp/portal/misc/partials/circle-button.html]: How does the optional-ness of `trunclen` work?

Comment (timlevett) [inline src/main/webapp/portal/misc/partials/circle-button.html]: Had a similar thought

Comment (jhanstra) [inline src/main/webapp/portal/misc/partials/circle-button.html]: If `data-trunclen` is not given then it doesn't truncate and shows the full text. Tested with the list-of-links widget.

Comment (timlevett) [inline src/main/webapp/portal/misc/partials/circle-button.html]: cool

**Comment by timlevett** [comment]
:+1:

**Pull request merged**